apps/meteor/tests/unit/server/modules/streamer/streamer.module.spec.ts

```
@@ -0,0 +1,131 @@
+describe('Streamer.sendToManySubscriptions', () => {
+	const createdStreamers: string[] = [];
+
+	const createStreamer = (): TestStreamer => {
+		const name = `streamer-test-${Date.now()}-${Math.random()}`;
+		createdStreamers.push(name);
+		return new TestStreamer(name);
+	};
+
+	afterEach(() => {
+		sinon.restore();
+		createdStreamers.splice(0).forEach((name) => delete StreamerCentral.instances[name]);
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)